src/Connection_Oracle.php

```diff
@@ -25,6 +25,36 @@ public function __construct($properties = [])
         parent::__construct($properties);
 
         // date and datetime format should be like this for Agile Data to correctly pick it up and typecast
-        $this->expr('ALTER SESSION SET NLS_DATE_FORMAT = {format}', ['format'=>'YYYY-MM-DD HH24:MI:SS'])->execute();
+        $this->expr('ALTER SESSION SET NLS_DATE_FORMAT={format} NLS_NUMERIC_CHARACTERS={dec_char}', [
```

Comment:
does {format} give you a proper escape? you know those are different between [format] and {format}